electron/src/renderer/realm/openRealm.ts

```diff
@@ -0,0 +1,46 @@
+import Realm from 'realm';
+import { Car } from 'schemas';
+import { ipcRenderer } from 'electron';
+import path from 'path';
+import { cloneDeep } from 'lodash';
+import app from './realmApp';
+import Authentication from './Authentication';
+
+async function getUserDataPath() {
+  const userDataPath = await ipcRenderer.invoke('get-user-data-path');
+  return userDataPath;
+}
+
+async function openRealm(): Promise<Realm | undefined> {
+  await Authentication.logIn();
+  const userDataPath = await getUserDataPath();
+  console.log('curr user', app.currentUser);
+  const userClone = cloneDeep(app.currentUser);
+  console.log('user clone', userClone);
```

Comment:
this can be removed i think. was part of attempt at different implementation that didn't work